docs/api_reference/scripts/custom_formatter.py

```diff
@@ -17,15 +17,16 @@ def process_toc_h3_elements(html_content: str) -> str:
 
     # Process each element
     for element in toc_h3_elements:
-        element = element.a.code.span
-        # Get the text content of the element
-        content = element.get_text()
+        if element.a.code:
```

Comment:
Pyoc with multiple levels of headings create `toc-h3` elements (even if there's no `.. contents` directive)